050-AIS-APIManagementwithFunctions/Coach/README.md

```diff
@@ -0,0 +1,85 @@
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+<!-- REMOVE_ME ${challengesSection} (remove this from your MD files if you are writing them manually, this is for the automation script) REMOVE_ME -->
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+-  Challenge 0: **[Prepare your Development Environment](Solution-00.md)**
+   - Get yourself ready to build your integration solution
+-  Challenge 1: **[Provision an Integration Environment](Solution-01.md)**
+   - Create a bicep template that will provision a baseline integration environment.
+-  Challenge 2: **[Deploy your Integration Environment](Solution-02.md)**
+   - Create a CI/CD pipeline to do automated deployment of your integration environment.
+-  Challenge 3: **[Create Backend APIs](Solution-03.md)**
+   - Create backend APIs
+-  Challenge 4: **[Secure Backend APIs](Solution-04.md)**
+   - Securing backend APIs by configuring them in the VNET or by using OAuth 2.0 authorization
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Coach Prerequisites 
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the \`/Student/Resources\` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.  
+
+### Additional Coach Prerequisites 
+- Visual Studio Code 1.63 or higher with Bicep extension
+- Azure CLI 2.20.0 or higher
+- Azure DevOps project or GitHub project
+
+## Azure Requirements
+
+This hack requires students to have access to an Azure subscription where they can create and consume Azure resources. These Azure requirements should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students.
+
```

Comment:
What are the Azure requirements for this hack?  What resources will a student consume?  What permissions will they need on the subscription? (Owner? Contributor? ReadOnly?)

This information is important to share with the customer ahead of time if you will run the hack in their subscription.

Reply:
Are we referring to the pre-requisites defined in Student/Challenge-00?   Do you want me to just copy those over into this file?

Reply:
Done - added new information on the requirements needed for the Azure subscription.

Reply:
Done

Reply:
Done